Kin Unity/Assets/Kin/Core/NativeBinding/NativeBridge.iOS.cs

```diff
@@ -92,8 +92,8 @@ public void GetMinimumFee( string clientId )
 			_kinGetMinimumFee( clientId );
 		}
 
-        [DllImport("__Internal")]
-        static extern void _kinRestoreAccount(string clientId);
+        //[DllImport("__Internal")]
```

Comment:
is this intentional? if not needed don't afraid to delete, it's more confusing to leave it.

Reply:
Its needed for backup and restore, but since its not implemented yet on iOS i commented it out